scripts/split-e2e-tests-v2.ts

```diff
@@ -155,7 +155,7 @@ const TEST_EXCLUSIONS: { l: string[]; w: string[] } = {
     'src/__tests__/pinpoint/notifications-pinpoint-config-util.ts',
   ],
 };
-const MAX_WORKERS = 3;
+const MAX_WORKERS = 4;
```

Comment:
can we do something like this for the base.yml and config.yml file?

Reply:
Not worth the effort given timeline for this script.